rpa-builder/6.0/modules/ROOT/pages/running-workflows-and-using-workflow-run-results.adoc

```diff
@@ -0,0 +1,55 @@
+= Running Workflows and Viewing Workflow Run Results
+
+To perform a test run of a Workflow in RPA Builder, click *Run Workflow* (image:run-process.png[The Run Workflow icon, 2%, 2%]) on the Workbench toolbar or press _F7_ on your keyboard.
+
+The RPA Builder window is hidden while the Workflow executes, so the engine can now use the mouse and keyboard just like a normal user would and you can watch to confirm if the Workflow performs correctly. Do not use the mouse or keyboard while the Workflow is running.
+
+Depending on the settings you made for the *Show Message boxes* value in the _Project Properties_, additional information may appear during the run. Note that every message box takes a few seconds to disappear, which causes a corresponding pause in the Workflow and affects measured times and Workflow stability.
+
+Before using a Workflow for productive measurement, turn off debug message boxes and perform at least one test run afterward.
```

Comment:
Can you just say "turn off debug messages" --if you can ditch box, also do so on line 7